### ISSUE TITLE
[Proposal] - Move season GUIDs out of main.py to allow easy future adjustment each season without recompiling

### ISSUE THREAD

**Issue by bbeenn1** [AnthonyMichaelTDM/DRG-Save-Editor#16]
Likely simple solution: 
Add season guids to `guids.json`
Will require some re-coding of main.py to load them correctly most likely.

Alternatively create another json to load them from such as `season_guids.json`.
This would lead to the ability to create another  `cosmetics_guids.json` as well if that is planned.

**Comment by AnthonyMichaelTDM** [comment]
I completely agree, I'm planning on refactoring the entire code base to follow SOLID design principles but I'm really busy with college right now.
